Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/qrcode/consent/SubmissionConsentViewModel.kt

```diff
@@ -32,6 +33,7 @@ class SubmissionConsentViewModel @AssistedInject constructor(
         launch {
             try {
                 val preAuthorized = tekHistoryProvider.preAuthorizeExposureKeyHistory()
+                if (preAuthorized) analyticsKeySubmissionCollector.reportAdvancedConsentGiven()
```

Comment:
AFAIK 

> advancedConsentGiven set to true if the user agreed to share keys when scanning the QR code and the user has not withdrawn the consent; false otherwise.

is about the legal consent, i.e.

![Screenshot from 2021-02-26 16-55-02](https://user-images.githubusercontent.com/1439229/109323112-63fbd800-7853-11eb-905e-cb9930da4de4.png)

not about the technical consent  to access the TEKs.

ping @mlenkeit 
cc @mtwalli

Reply:
@d4rken is right. Sorry for not being particularly clear in the spec. When it was written, the "technical consent" didn't even exist I think 😄